src/MailHide.php

```diff
@@ -273,11 +303,11 @@ public function getHtml($email = null)
             throw new MailHideException('Missing email address');
         }
 
-        if ($this->publicKey === null) {
+        if ($this->getPublicKey() === null) {
```

Comment:
MailHide is still on v1's API which uses the nomenclature "public_key" and "private_key".